src/invaders.rs

```diff
@@ -9,6 +9,8 @@ pub struct Invader {
     pub x: usize,
     pub y: usize,
     points: u16,
+    face1: &'static char,
+    face2: &'static char,
```

Comment:
There's no benefit to using a reference here. `char` is a `Copy` type, and it's just 32 bits. A reference creates a 64-bit pointer (or 32-bit if you have a 32-bit processor) to the location of 32 bits, requiring extra time to retrieve, extra space to store, and extra syntax to deal with.

```suggestion
    face1: char,
    face2: char,
```